CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # CHANGELOG
 
-## Next Release
+## v7.6.0 (2024-10-01)
```

Comment:
should be a patch version since it's only fixing something and not adding a new feature

Reply:
Figured the removed function was worth a minor bump to hopefully cause caution for those updating.

Reply:
That would improperly classify what's actually happening here though, those that have their libs pinned to a patch version won't get this update by doing a minor bump.

There are only a dozen customers using this feature behind a feature flag, all were contacted. It's also been removed today from the API so regardless of us removing it in the lib, it's no longer available to consume. At this point, it's a chore of removing dead code since the endpoint itself no longer exists.